Add component suffix to generator (#111)
github/actionview-component: README.md

Content passed to an `ActionView::Component` as a block is captured and assigned to the `content` accessor.

#### Quick start

> Use the provided component generator to quickly get going with a new `ActionView::Component`.

Reviewer: ```suggestion
Use the component generator to create a new `ActionView::Component`.
```

> The generator expects the component name and the list of accepted properties as arguments. 

Reviewer: ```suggestion
The generator accepts the component name and the list of accepted properties as arguments:
```

```bash
bin/rails generate component Example title content
      invoke  test_unit

>       create    test/components/example_component_test.rb

Reviewer: ```suggestion
      create  test/components/example_component_test.rb
```
